server/aurora_mcp/chat_bridge.py

```
@@ -0,0 +1,252 @@
+
+
+async def _poll_step(
+    api_call: ApiCall, sid: str, last_seq: int,
+) -> Optional[Tuple[Dict[str, Any], List[Dict[str, Any]], str, int]]:
+    """One poll iteration. Returns None on transient error so the caller retries."""
+    try:
+        return await _poll_once(api_call, sid, last_seq)
+    except (asyncio.TimeoutError, TimeoutError):
+        logger.warning("chat_with_aurora poll timed out, retrying (session=%s)", sid)
+        return None
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)